setup.py

```diff
@@ -24,7 +24,8 @@ def get_meta():
 
 install_requires = [
     'celery',
-    'falcon',
+    'types-requests',
+    'falcon==2.0.0',
```

Comment:
the other pr that ive saw,,, is that this requirements was placed on the docker file could you change that like this also its added on the setup.py or requirements.txt

Reply:
As @ninotoshi mentioned the other PR would be unnecessary as the error comes from this parsely module.